gateway/src/lib.rs

```diff
@@ -109,7 +109,6 @@
 #![allow(clippy::module_name_repetitions, clippy::must_use_candidate)]
 
 pub mod cluster;
-pub mod queue;
```

Comment:
Export the crate as `queue` or equivalent so that the root isn't polluted and there's no breaking change